src/Common/FitData.m

```diff
@@ -77,6 +77,7 @@
 
     % Find voxels that are not empty
     if isfield(data,'Mask') && (~isempty(data.Mask))
+        data.Mask(isnan(data.Mask))=0;
```

Comment:
This seems quite reasonable to me. Would saving `isnan(data.Mask)` output be extra?